### PR TITLE
Add compat data for Presentation

### PR DESCRIPTION
**Pull request by LarsBaertschi**
I am not really sure what standard_track for this should be, because it is only a candidate recommendation.

### CODE REVIEW

File: api/Presentation.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/Presentation.json]: This should be `true`.

File: api/PresentationAvailability.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationAvailability.json]: ditto

File: api/PresentationConnection.json [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationConnection.json]: ditto

File: api/PresentationConnection.json [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationConnection.json]: ditto

File: api/PresentationConnection.json [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
```
Comment (ExE-Boss) [inline api/PresentationConnection.json]: ditto

File: api/PresentationConnection.json [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
```
Comment (ExE-Boss) [inline api/PresentationConnection.json]: ditto

File: api/PresentationConnectionAvailableEvent.json [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationConnectionAvailableEvent.json]: ditto

File: api/PresentationConnectionCloseEvent.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationConnectionCloseEvent.json]: ditto

File: api/PresentationConnectionClosedEvent.json [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationConnectionClosedEvent.json]: ditto

File: api/PresentationConnectionList.json [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationConnectionList.json]: ditto

File: api/PresentationReceiver.json [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationReceiver.json]: ditto

File: api/PresentationRequest.json [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationRequest.json]: ditto

File: api/PresentationRequest.json [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (ExE-Boss) [inline api/PresentationRequest.json]: ditto

File: api/PresentationConnection.json [hunk 14]
```diff
@@ -0,0 +1,211 @@
...
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "events": {
```
Comment (teoli2003) [inline api/PresentationConnection.json]: There should be one entry for each event handler properties.

File: api/PresentationConnection.json [hunk 15]
```diff
@@ -0,0 +1,211 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PresentationConnection/close",
```
Comment (teoli2003) [inline api/PresentationConnection.json]: The url should end in /onXXX

File: api/PresentationConnectionCloseEvent.json [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
```
Comment (teoli2003) [inline api/PresentationConnectionCloseEvent.json]: Constructors, properties, and methods should be added as sub-features.

File: api/PresentationConnectionClosedEvent.json [hunk 17]
```diff
@@ -0,0 +1,55 @@
...
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
```
Comment (teoli2003) [inline api/PresentationConnectionClosedEvent.json]: Constructors, properties, and methods should be added as sub-features.

File: api/PresentationConnectionList.json [hunk 18]
```diff
@@ -0,0 +1,55 @@
...
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
```
Comment (teoli2003) [inline api/PresentationConnectionList.json]: Constructors, properties, and methods should be added as sub-features.

File: api/PresentationReceiver.json [hunk 19]
```diff
@@ -0,0 +1,55 @@
...
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
```
Comment (teoli2003) [inline api/PresentationReceiver.json]: Constructors, properties, and methods should be added as sub-features.

File: api/PresentationRequest.json [hunk 20]
```diff
@@ -0,0 +1,107 @@
...
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
```
Comment (teoli2003) [inline api/PresentationRequest.json]: Constructors, properties, and methods should be added as sub-features.

File: api/PresentationConnectionAvailableEvent.json [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
```
Comment (teoli2003) [inline api/PresentationConnectionAvailableEvent.json]: Constructors, properties, and methods should be added as sub-features.

File: api/PresentationConnection.json [hunk 22]
```diff
@@ -0,0 +1,211 @@
...
+            "samsunginternet_android": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
```
Comment (teoli2003) [inline api/PresentationConnection.json]: Constructors, properties, and methods should be added as sub-features.

File: api/PresentationAvailability.json [hunk 23]
```diff
@@ -0,0 +1,55 @@
...
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
```
Comment (teoli2003) [inline api/PresentationAvailability.json]: Constructors, properties, and methods should be added as sub-features.

File: api/Presentation.json [hunk 24]
```diff
@@ -0,0 +1,55 @@
...
+          "samsunginternet_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
```
Comment (teoli2003) [inline api/Presentation.json]: Constructors, properties, and methods should be added as sub-features.

**Pull request merged**